service-loadbalancer/service_loadbalancer.go

```diff
@@ -110,6 +111,13 @@ var (
 
 	startSyslog = flags.Bool("syslog", false, `if set, it will start a syslog server
 		that will forward haproxy logs to stdout.`)
+
+	defStaticPage = flags.String("static-page", "/etc/haproxy/errors/404.http", `if set,
```

Comment:
hmm, do we really need 2 flags? given this is a file on disk, no point opening it as a flag right? can we just check the errorPage, and if it isn't set or returns a non 200, always use the 404.http page? I've found that the number of states in the system roughly doubles with each new flag, because someone will try overloading it to mean something else down the line, so if we can get rid of this flag please do so.